.gitignore

```diff
@@ -35,9 +38,6 @@ lerna-debug.log
 # Debug logs from yarn
 **/yarn-error.log
 
-# IntelliJ IDE ignore
-.idea
-
 .DS_Store
```

Comment:
I think you can remove this too, OSX users should know to ignore this in their dotfiles.